src/router/handlers/user.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @swagger
+ * /user/:
+ *  get:
+ *    description: Use to request a user
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        schema:
+ *          type: object
+ *          properties:
+ *            user:
+ *              type: object
+ */
+let getUser = async (req, res) => {
+    
+};
+
+/**
+ * @swagger
+ * /user/:
+ *  put:
+ *    description: Use to update a user
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have user privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let updateUser = async (req, res) => {
+    
+};
+
+/**
+ * @swagger
+ * /user/:
+ *  delete:
+ *    description: Use to delete a user
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have user privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let deleteUser = async (req, res) => {
+    
+};
```

Comment:
Need a way to `createUser` with `post`

Reply:
That is an auth endpoint I forgot to implement actually.